python/pyspark/ml/linalg/__init__.py

```diff
@@ -386,14 +386,14 @@ def squared_distance(self, other):
 
     def toArray(self):
         """
-        Returns an numpy.ndarray
+        Returns the underlying numpy.ndarray
```

Comment:
I think this is OK, but just wondering how much to promise that it's the underlying array

Reply:
In case of `DenseVector` it means that there is a vector containing the values...and it think makes sense to report it to the users, in order that they know what happens if they modify it...